### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by traustitj** [PiranhaCMS/piranha.core#1076]
I am trying to add a user page that requires authentication. So I added a user with rights to view pages, and nothing else, but he isn't able to login, at all. You are just redirected to login page again, no error.
As far as I have tested, you have to be administrator to login.

The user should be able to login, and ONLY see his own information, something I will make.

Is this possible?

**Comment by tidyui** [comment]
No. The minimum requirements for logging into the admin interface is that the user is an admin. It hasn’t been designed with that use case in mind.

**Comment by traustitj** [comment]
Thank you for the quick reply. Do you have any instructions on how to use the admin to go to authorized pages? Or how to do best practice of authorized pages?

**Comment by tidyui** [comment]
I think we will have to do a tutorial on it, but in short, here are my suggestions:

1. Add `AuthorizeAttribute` on manager pages.
2. Do **not** add `AuthorizeAttribute` on public pages, instead select the claims the user should have on the page instance.
3. For public pages, get them using the `IModelLoader`, like in the template projects. The loader will perform security checks for the current user and the requested page.
4. If the user doesn't have access, the `IModelLoader` will throw a `UnathorizedException`, customize the middleware pipeline to redirect users when this happen to a **public** login page, not the login page of the manager.

Best regards

**Comment by traustitj** [comment]
I don't understand you here

**Comment by tidyui** [comment]
We will provide a tutorial for you

**Comment by tidyui** [comment]
Please take a look at this tutorial and let us know if it makes sense or should be improved in any way!

Source code:
https://github.com/PiranhaCMS/piranha.core.docs/blob/master/src/08.%20Tutorials/01.%20Securing%20Pages.md

Webpage:
http://piranhacms.org/docs/tutorials/securing-pages

Regards
